README.md

```diff
@@ -27,11 +27,19 @@ individual tasks.
 
 # Installation
 
-Run is a single binary, which you can download from from the [releases
-page][releases].
+Run is a single binary, which you can download from from the [releases page][releases].
 
-Alternately, if you already use go, you can install Run with the go command
-line tool:
+## Install Script (MacOS, Linux)
+
+This command retrieves and executes the shell installation script for the latest version of run, installing it into the current working directory.
```

Comment:
```suggestion
This command can be used to fetch the install script and execute it in the current working directory.
```

Reply:
I'm not sure which of these is better?

Reply:
"fetching the script" feels like an implementation detail to me--it's confusing for casual readers, and already obvious to sophisticated ones

how about, "This install command will place download the latest version of run for your platform and place it in the current directory"